assets/panel/debugger.properties

```diff
@@ -452,7 +452,7 @@ editor.singleResult=1 result
 
 # LOCALIZATION NOTE (editor.noResults): Editor Search bar message
 # for when no results found.
-editor.noResults=No results
+editor.noResults=No results found
```

Comment:
> If your changes are relevant only for English — for example, to correct a typographical error or to make letter case consistent — then there is generally no need to update the entity name.

I just appended "found" keyword which doesn't change the meaning still if it requires new string ID let me know. @flodolo

Reply:
It might not change the general meaning, but the resulting translations will be likely different. I'd still change the ID, if it's important to update the string for en-US

Reply:
Added new string with id : "editor.noResultsFound" and value : "No results found"